frontends/main/src/app/healthcheck/route.ts

```diff
@@ -0,0 +1,3 @@
+export async function GET() {
+  return Response.json({ status: "ok" }, { status: 200 })
+}
```

Comment:
BTW: This is a NextJS route handler file for making APIs. We don't use these much, since our APIs are all on Django.

(The only other one we have is for sitemap index)

https://nextjs.org/docs/app/getting-started/route-handlers-and-middleware